data/json/items/comestibles/med.json

```diff
@@ -1288,5 +1288,54 @@
         "addiction_potential" : 40,
         "addiction_type" : "sleeping pill",
         "use_action" : "XANAX"
+    },
+    {
+        "id" : "disinrag",
+        "type" : "COMESTIBLE",
```

Comment:
Suggest adding `"copy-from":` to condense as well as any fallbacks if the original items are altered.